force_wfmanager/wfmanager_global_task.py

```diff
@@ -12,6 +13,18 @@
 
 log = logging.getLogger(__name__)
 
+IS_WINDOWS = (platform.system() == "Windows")
+
+
+class ExitAction(TaskAction):
+    """ A standard 'Quit' (Mac) or 'Exit' (Windows) menu action. """
+
+    accelerator = "Alt+F4" if IS_WINDOWS else "Ctrl+Q"
+    method = "setup_task.exit"
+
+    def _name_default(self):
+        return u"Exit" if IS_WINDOWS else u"Quit"
+
```

Comment:
Couldn't find much doc on ApplicationTask, so couldn't figure out how to implement that. But `set_task.exit` now calls the application's exit method.